training/models/training_material.py

```diff
@@ -0,0 +1,44 @@
+# -*- coding: utf-8 -*-
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+# © 2015 Grupo ESOC Ingeniería de Servicios, S.L.U. - Jairo Llopis
+
+from openerp import fields, models
+from .common import M
+
+
+class MaterialType(models.Model):
+    """Type of the training materials."""
+    _name = M % "material_type"
```

Comment:
I'm more on the mood of the `module.model` schema, same as views. Seems more homogeneous to me. Maybe other community members can opine.